tests/timer.rs

```diff
@@ -71,6 +71,7 @@ fn starving() {
     rx.recv().unwrap();
 }
 
+#[allow(deprecated)]
```

Comment:
Could you remove this? I assume you get a warning. We should fix this in a follow up PR by updating the test.

Reply:
I removed the commit which had this.